src/Ens/JobeetBundle/DataFixtures/ORM/LoadCategoryData.php

```diff
@@ -0,0 +1,44 @@
+<?php
+// src/Ens/JobeetBundle/DataFixtures/ORM/LoadCategoryData.php
+namespace Ens\JobeetBundle\DataFixtures\ORM;
+ 
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Ens\JobeetBundle\Entity\Category;
+ 
+class LoadCategoryData extends AbstractFixture implements OrderedFixtureInterface
+{
+  public function load(ObjectManager $em)
+  {
+    $design = new Category();
+    $design->setName('Design');
+ 
+    $programming = new Category();
+    $programming->setName('Programming');
+ 
+    $manager = new Category();
+    $manager->setName('Manager');
+ 
+    $administrator = new Category();
+    $administrator->setName('Administrator');
+ 
+    $em->persist($design);
+    $em->persist($programming);
+    $em->persist($manager);
+    $em->persist($administrator);
+ 
+    $em->flush();
+ 
+    $this->addReference('category-design', $design);
+    $this->addReference('category-programming', $programming);
+    $this->addReference('category-manager', $manager);
+    $this->addReference('category-administrator', $administrator);
+  }
+ 
+  public function getOrder()
+  {
+    return 1; // the order in which fixtures will be loaded
+  }
+}
+?>
```

Comment:
@PhotonSapiens 
please never close php tag
This is some very old legacy from the time when PHP was for **P**HP **H**ypertext **P**reprocessor which is no longer true - now PHP is fully-featured programming language and PHP code usually takes a whole file, without mixing up with HTML templating, so there's no need to close tag
Closing it may also be harmful - if you will add line break after closing tag you wil not notice, but inclusion of this file will produce extra line break in output an you will spend ages figuring out where this line break comes from